--- Makefile.am.old	2009-11-06 06:09:01.395463173 -0500
+++ Makefile.am	2009-11-06 06:10:44.367608706 -0500
@@ -45,6 +45,7 @@
 
 tora_SOURCES=main.cpp \
 	tomainwindow.cpp tomainwindow.h \
+	cregistry.cpp cregistry.h \
 	toabout.cpp toabout.h \
 	tobackground.cpp tobackground.h \
 	tobackgroundlabel.cpp tobackgroundlabel.h \
@@ -332,6 +333,7 @@
 	moc_tolinechart.cpp \
 	moc_tomain.cpp \
 	moc_tomainwindow.cpp \
+	moc_cregistry.cpp \
 	moc_tomarkedtext.cpp \
 	moc_tomemoeditor.cpp \
 	moc_tomodeleditor.cpp \
