diff --exclude debian -ruN tora_1.3.20.orig/toglobalsetting.cpp tora-1.3.20/toglobalsetting.cpp
--- tora_1.3.20.orig/toglobalsetting.cpp	2005-04-19 12:30:42.000000000 +0200
+++ tora-1.3.20/toglobalsetting.cpp	2006-02-11 13:23:54.000000000 +0100
@@ -84,7 +84,7 @@
     HistorySize->setValue(toTool::globalConfig(CONF_STATUS_SAVE,
                           DEFAULT_STATUS_SAVE).toInt());
     IncludeDB->setChecked(!toTool::globalConfig(CONF_DB_TITLE, "Yes").isEmpty());
-    MaximizeMain->setChecked(!toTool::globalConfig(CONF_MAXIMIZE_MAIN, "Yes").isEmpty());
+    MaximizeMain->setChecked(!toTool::globalConfig(CONF_MAXIMIZE_MAIN, "").isEmpty());
     Statusbar->setChecked(!toTool::globalConfig(CONF_MESSAGE_STATUSBAR, "").isEmpty());
     RestoreSession->setChecked(!toTool::globalConfig(CONF_RESTORE_SESSION, "").isEmpty());
     HelpDirectory->setText(toHelpPath());
diff --exclude debian -ruN tora_1.3.20.orig/tomain.cpp tora-1.3.20/tomain.cpp
--- tora_1.3.20.orig/tomain.cpp	2005-06-28 22:29:35.000000000 +0200
+++ tora-1.3.20/tomain.cpp	2006-02-11 13:24:17.000000000 +0100
@@ -484,7 +484,7 @@
         TOCATCH
     }
 
-    if (!toTool::globalConfig(CONF_MAXIMIZE_MAIN, "Yes").isEmpty() && Connections.empty() )
+    if (!toTool::globalConfig(CONF_MAXIMIZE_MAIN, "").isEmpty() && Connections.empty() )
         showMaximized();
     else
         show();
 
